pkg/api/api.go

```diff
@@ -21,11 +21,14 @@
 //
 // SecurityDefinitions:
 // basic:
-//  type: basic
+//
+//	type: basic
+//
```

Comment:
this should be valid yaml otherwise the `make swagger-api-spec` fails with:
```
yaml: unmarshal errors:
  line 4: mapping key "type" already defined at line 2
```
```suggestion
//  type: basic
```

Reply:
I can fix it by making all lines in the yaml indented, but then I get

```
SWAGGER_GENERATE_EXTENSION=false /Users/sakjur/go/bin/swagger-v0.30.2 generate spec -m -w pkg/server -o public/api-spec.json \
	-x "github.com/grafana/grafana/pkg/services/ngalert/api/tooling/definitions" \
	-x "github.com/prometheus/alertmanager" \
	-i pkg/api/swagger_tags.json
yaml: line 2: mapping values are not allowed in this context
make: *** [--swagger-api-spec] Error 1
```

We can't really _not_ upgrade, fwiw, so unless we'll find a good solution to making Swagger work for v9.1.x, I'd suggest breaking Swagger for the v9.1.x branch (it should be finalized for that version around the time of v9.1.0, so it wouldn't be the weirdest of things to do)

Reply:
I'm afraid breaking swagger is not an option unless [build-go](https://github.com/grafana/grafana/blob/f7a3e50b23b6b4e5a40041996fd267646a313ec7/Makefile#L75) target no more depends on that

however I think that the following is OK with `gofmt`: can you try it? 🙏 
```
// Package api Grafana HTTP API.
//
// The Grafana backend exposes an HTTP API, the same API is used by the frontend to do
// everything from saving dashboards, creating users and updating data sources.
//
// Schemes: http, https
// BasePath: /api
// Version: 0.0.1
// License: GNU Affero General Public License v3.0 https://www.gnu.org/licenses/agpl-3.0.en.html
// Contact: Grafana Labs<hello@grafana.com> https://grafana.com
//
//	Consumes:
//	- application/json
//
//	Produces:
//	- application/json
//
//	Security:
//	- basic:
//	- api_key:
//
//	SecurityDefinitions:
//	basic:
//		type: basic
//	api_key:
//		type: apiKey
//		name: Authorization
//		in: header
//
// swagger:meta
```

Reply:
this is how it's in main and seems to be OK both with `gofmt` and `swagger`
https://github.com/grafana/grafana/blob/main/pkg/api/api.go#L1-L30

Reply:
I have pushed [some changes](https://github.com/grafana/grafana/pull/55733/commits/372e9cab0bf57886bfa87c808076a79a0dc37950) that work for me locally

Reply:
🙏 Thank you!